op-supernode/supernode/types/types.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+import "fmt"
+
+type ChainID uint64
```

Comment:
We already have `eth.ChainID` which would be better to reuse.  It's not a uint64 because it's designed to support the full range of possible chain IDs (people are still debating if uint64 is enough chain IDs for anyone...), but it does have a `UInt64()` getter.  Would definitely be nice to centralise on a single ChainID type.